web/libs/datamanager/src/components/CellViews/Agreement/Agreement.scss

```diff
@@ -0,0 +1,19 @@
+.agreement {
+  &__score {
+    &_zero {
+      color: #000;
+    }
+
+    &_low {
+      color: #DD0000;
+    }
+
+    &_medium {
+      color: #D98F00;
+    }
+
+    &_high {
+      color: #419400;
+    }
```

Comment:
Flagging these for further review. We don't necessarily have to do anything about these just yet but I want to start raising any hardcoded colors that block us from theming. Part of these conversion efforts thus far seems very inline of making massive increments towards better CSS management across the board and utilization of design tokens. If we take these opportunities to address what we can, itemize the ones which we cannot field just yet, this will have a massive impact to the end result of these initiatives.
```
    &_low {
      color: #DD0000;
    }

    &_medium {
      color: #D98F00;
    }

    &_high {
      color: #419400;
    }
```